packages/component-library/stories/Modal.stories.tsx

```diff
@@ -626,8 +626,10 @@ TestScrollingModalAndScrollingContent.story = {
 loadElmStories("Modal (Elm)", module, require("./ModalStories.elm"), [
   "Confirmation (cautionary)",
   "Generic",
-  "Confirmation (Informative)",
-  "Confirmation (Positive)",
-  "Confirmation (Negative)",
-  "Confirmation (User action)",
+  "Confirmation (informative)",
+  "Confirmation (positive)",
+  "Confirmation (negative)",
+  "Confirmation (user action)",
+  "InputEdit (positive)",
+  "InputEdit (negative)",
```

Comment:
I added a suggestion in Slack to add "auto open" to the name of the stories where they auto open (and the React ones too for consistency)... waiting to hear back.

Reply:
We can change the story names in a different PR, so don't worry about this.